internal/command/show.go

```diff
@@ -470,15 +506,19 @@ func getStateFromPath(path string, enc encryption.Encryption) (*statefile.File,
 }
 
 // getStateFromBackend returns the State for the current workspace, if available.
-func getStateFromBackend(b backend.Backend, workspace string) (*statefile.File, error) {
+func getStateFromBackend(ctx context.Context, b backend.Backend, workspace string) (*statefile.File, error) {
+	_, span := tracing.Tracer().Start(ctx, "Get State from Backend")
+	defer span.End()
 	// Get the state store for the given workspace
-	stateStore, err := b.StateMgr(workspace)
+	stateStore, err := b.StateMgr(context.TODO(), workspace)
```

Comment:
First, why ignoring `ctx` from `tracing.Tracer().Start()`? And why using `context.TODO()` here?

Reply:
Mistake, fixed!